### ISSUE TITLE
help

### ISSUE THREAD

**Issue by NA7KR** [sfjohnso/PicoW#1]
Hello, I was following picscopr in Makezine and loaded firmware and set all options,
I uploaded PicoScope.bas with XMODEM and the screen is blank.

Do need to upload all files? any order?

Kevin

**Comment by sfjohnso** [comment]
Hi, Kevin - Thank you for trying out my code!  It really makes me happy when people do so.Please be sure that all of the options are set according to the steps specified in the article.  You should see confirmation via the terminal emulator after each (this may only be a momentary pause, then the “>” prompt).If you are still having issues, please let me know (there are some test commands that you can run to verify that the LCD is working properly; they’re discussed in the MMBasic manual, which I don’t have with me right now).Best regards - - SteveOn Mar 3, 2024, at 3:25 PM, NA7KR ***@***.***> wrote:﻿
Hello, I was following picscopr in Makezine and loaded firmware and set all options,
I uploaded PicoScope.bas with XMODEM and the screen is blank.
Do need to upload all files? any order?
Kevin

**Comment by NA7KR** [comment]
Hi Steve,

I found the answer was missing the run command. Is there a way to start it running automatically?

 

Thanks, Kevin

NA7KR

**Comment by sfjohnso** [comment]
Yes!  Use the Flash Overwrite 1 command to save the code to the flash area, then the Autorun 1 command to get it to automatically start. You can find the precise syntax in the MMBasic manual; unfortunately I don’t have a copy with me at the moment.Best regards - - SteveOn Mar 4, 2024, at 9:41 AM, NA7KR ***@***.***> wrote:﻿
Hi Steve,



I found the answer was missing the run command. Is there a way to start it running automatically?



 



Thanks, Kevin



NA7KR



 



 



From: Steve Johnson ***@***.***> 

Sent: Sunday, March 3, 2024 1:49 PM

To: sfjohnso/PicoW ***@***.***>

Cc: NA7KR ***@***.***>; Author ***@***.***>

Subject: Re: [sfjohnso/PicoW] help (Issue #1)



 



Hi, Kevin - Thank you for trying out my code! Â It really makes me happy when people do so.Please be sure that all of the options are set according to the steps specified in the ar



 ‌ ‌ ‌ ‌ ‌ ‌ ‌ ‌ ‌  ‌ ‌ ‌ ‌ ‌ ‌ ‌ ‌ ‌ ‌  ‌ ‌ ‌ ‌ ‌ ‌ ‌ ‌ ‌  ‌ ‌ ‌ ‌ ‌ ‌ ‌ ‌ ‌ ‌  ‌ ‌ ‌ ‌ ‌ ‌ ‌ ‌ ‌  ‌ ‌ ‌ ‌ ‌ ‌ ‌ ‌ ‌ ‌  ‌ ‌ ‌ ‌ ‌ ‌ ‌ ‌ ‌  ‌ ‌ ‌ ‌ ‌ ‌ ‌ ‌ ‌ ‌  ‌ ‌ ‌ ‌ ‌ ‌ ‌ ‌ ‌  ‌ ‌ ‌ ‌ ‌ ‌ ‌ ‌ ‌ ‌  ‌ ‌ ‌ ‌ ‌ ‌ ‌ ‌ ‌  ‌ ‌ ‌ ‌ ‌ ‌ ‌ ‌ ‌ ‌  ‌ ‌ ‌ ‌ ‌ ‌ ‌ ‌ ‌  ‌ ‌ ‌ ‌ ‌ ‌ ‌ ‌ ‌ ‌  ‌ ‌ ‌ ‌ ‌ ‌ ‌ ‌ ‌  ‌ ‌ ‌ ‌ ‌ ‌ ‌ ‌ ‌ ‌  ‌ ‌ ‌ ‌ ‌ ‌ ‌ ‌ ‌  ‌ ‌ ‌ ‌ ‌ ‌ ‌ ‌ ‌ ‌  ‌ ‌ ‌ ‌ ‌ ‌ ‌ ‌ ‌  





CAUTION: External email from: notifications@‌github.‌com

Use caution before clicking on links or opening attachments. 



	



Protection by  <https://mxguardian.net> MXGuardian 



Hi, Kevin - Thank you for trying out my code!  It really makes me happy when people do so.Please be sure that all of the options are set according to the steps specified in the article.  You should see confirmation via the terminal emulator after each (this may only be a momentary pause, then the “>” prompt).If you are still having issues, please let me know (there are some test commands that you can run to verify that the LCD is working properly; they’re discussed in the MMBasic manual, which I don’t have with me right now).Best regards - - SteveOn Mar 3, 2024, at 3:25 PM, NA7KR ***@***.*** <mailto:***@***.***> > wrote:﻿ 

Hello, I was following picscopr in Makezine and loaded firmware and set all options, 

I uploaded PicoScope.bas with XMODEM and the screen is blank. 

Do need to upload all files? any order? 

Kevin
